apps/definition_accept/test/support/fakes.ex

```diff
@@ -0,0 +1,38 @@
+defmodule FakeAcceptConnection do
+    defstruct connect: nil
+
+    def new!() do
+        struct(__MODULE__, %{})
+    end
+
+    defimpl Accept.Connection do
+        def connect(_accept, _opts) do
+        end
+    end
+end
+
+defmodule FakeDestination do
+    defstruct start_link: nil, write: nil, stop: nil, delete: nil
+
+    def new!(_overrides \\ []) do
```

Comment:
we could probably remove this since we're not using it.
```suggestion
    def new!() do
```

Reply:
Done